src/utils/plotSpecs/generateVolcanoSpec.js

```diff
@@ -62,28 +62,24 @@ const generateSpec = (configSrc, data) => {
         encode: {
           title: {
             update: {
-              fontSize: { value: 14 },
+              fontSize: 14,
             },
           },
           labels: {
-            interactive: true,
             update: {
               fontSize: { value: 12 },
               fill: { value: config.colour.masterColour },
             },
-            hover: {
-              fill: { value: 'firebrick' },
-            },
```

Comment:
Removed this as I noticed it here, in my previous PR #387 Oliver and I noticed a strange/misleading (because you can't actually click on them) hover effect that I forgot to remove earlier.